### DISCUSSION TITLE
Proper way to restart or shutdown Security Onion 2.3.61

### DISCUSSION THREAD

**Discussion by karanandcobi** [Security-Onion-Solutions/securityonion#4975] [Unsupported Versions] [answered]
Hello Everybody, I am quite new to Security Onion and running a Distributed Grid (1 Manager, 1 Search and 1 Sensor Nodes) in Air Gapped Environment. Recently I have updated Security Onion from 2.3.60 to 2.3.61. After that I am getting following message:

****************************************************************************************************
*  The following nodes in your Security Onion grid may need to be restarted due to package updates. * 

*  If the node has already been patched, restarted and been up for less than 15 minutes, then it          * 
* may not have updated it's restart_needed status yet. This will cause it to be listed below, even          * 
* if it has already been restarted. This feature will be improved in the future.                                        *

I have successfully executed "sudo salt-call state.highstate" on search and sensor node.

I understand to reboot the system at OS Level, "sudo reboot" can be used. but, my concern is potential data loss or damage to docker files. Can any body tell, what is the proper way to shutdown or reboot Security Onion at complete system level so that there is no risk of data loss or system corruption. 

Thank you in Advance.

**Comment by TOoSmOotH** [comment, marked as answer]
Issuing a standard reboot will not cause any issues.
